ichnaea/api/locate/views.py

```diff
@@ -94,3 +53,13 @@ def locate(self, api_key):
             api_key=api_key,
             api_name=self.view_name,
         ).search(query)
+
+    def prepare_location_data(self, location_data):
+        return location_data
```

Comment:
Can you stick a pragma no cover on this line, to get line coverage back to 100%? Not quite sure if this should be a raise NotImplementedError instead.